### PR TITLE
[2/X] Update feature/sql-refactor | Connection initialization

### PR DESCRIPTION
**Pull request by MarkL4YG**
## This PR
Smaller update to push the connection initialization.  
With this the ``TESqlManager`` class is ready for the migrations feature.

### CODE REVIEW

File: src/main/java/com/erigitic/sql/TESqlManager.java [hunk 1]
```diff
@@ -130,31 +153,38 @@ public Connection getJobsConnection() {
...
         accountsConnection = null;
 
+        suppressedCloseConnectionIfNotNull(jobsConnection, "Failed to close jobs connection!");
+        jobsConnection = null;
+
+        suppressedCloseConnectionIfNotNull(commandConnection, "Failed to close commands connection!");
+        commandConnection = null;
+    }
+
+    private void suppressedCloseConnectionIfNotNull(Connection accountsConnection, String s) {
```
Comment (Erigitic) [inline src/main/java/com/erigitic/sql/TESqlManager.java]: Should the `accountsConnection` variable be renamed to something more general such as `connection`?

Comment (MarkL4YG) [reply src/main/java/com/erigitic/sql/TESqlManager.java]: Not really as the ``accountsConnection`` is named that way in order to tell it should be used for querying account information such as balances or settings. Renaming it to just ``connection`` would falsely suggest a general purpose connection in my opinion.

Comment (Erigitic) [reply src/main/java/com/erigitic/sql/TESqlManager.java]: It seems weird though as when this function is called on lines 171, 174 and 177 it takes in arguments such as `accountsConnection`, `jobsConnection`, and `commandConnection` respectively. So it isn't only taking in an account connection.

Comment (MarkL4YG) [reply src/main/java/com/erigitic/sql/TESqlManager.java]: Oh I see. I was thinking you meant the entire variable+field and not that you're talking about that one method parameter ^^``  
In that case you're right oc. I'll get that changed.

**Pull request merged**